api/graphite.go

```diff
@@ -1025,14 +1040,45 @@ func (s *Server) graphiteTagFindSeries(ctx *middleware.Context, request models.G
 		return
 	default:
 	}
-	seriesNames := make([]string, 0, len(series))
-	for _, serie := range series {
-		seriesNames = append(seriesNames, serie.Pattern)
+
+	var warnings []string
+	if len(series) == limit {
+		warnings = append(warnings, "Result set truncated due to limit")
```

Comment:
this would also trigger if the amount of series happens to match the limit. maybe clusterFindByTag should explicitly set a flag when it needs to truncate.

Reply:
Yeah, I though about that, but I figured it was a reasonable edge case.

Reply:
i can live with that